apiary.apib

```diff
@@ -2881,6 +2881,66 @@ Update a milestone.
 
 + Response 204 (application/json;charset=utf-8)
 
+## Project items [/projectItems]
+
+Project items are added to a project and they affect the profitability. This information is only available for account administrators and project decision makers.
+
+*Required scopes: `projects`*
+
+### projectItems.list [GET /projectItems.list]
+
+Get a list of project items.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, required)
```

Comment:
I think this should be optional. That would mean it would be possible to retrieve all the project items for every projects you have access to (which is not a realistic use case), but I don't think that's a problem.

Reply:
Ok, but let's first see what the outcome of the meeting around this parameter is.

Reply:
Fixed